src/platform/web/ui/css/right-panel.css

```diff
@@ -0,0 +1,31 @@
+.RoomInfo {
+    grid-area: right;
+    flex-direction: column;
+}
+
+.RoomAvatar {
+    display: flex;
+}
+
+.RoomName h2 {
+    text-align: center;
+}
+
+.SidebarRow {
```

Comment:
Were you envisioning that this be used in other sidebars in the app? I'd prefer calling anything related to the right panel that, rather than sidebar. This is consistent with left panel and is more specific which panel is meant.